docs/api.md

```diff
@@ -973,6 +973,8 @@ Shortcut for [page.mainFrame.select()](#frameselectselector-values)
   - `sameSite` <[string]> `"Strict"` or `"Lax"`.
 - returns: <[Promise]>
 
+> **NOTE** For each cookie object, at least one of `url` and `domain` must be specified.
```

Comment:
@JoelEinbinder why is both `domain` and `url` needed? Can we fix this in protocol?